src/metabase/sync/sync_metadata/metabase_metadata.clj

```diff
@@ -54,7 +53,7 @@
                              :name   table-name
                              :active true)]
          (if field-name
-           (db/update-where! Field {:name field-name, :table_id table-id} k value)
+           (t2/update! Field {:name field-name, :table_id table-id} {k value})
```

Comment:
This is from before, but the docstring needs to be updated: "Returns `true` if a property was successfully set" is not true anymore.

Reply:
nice catch, I'll preserve the current behavior and wrap these update with `pos?`